src/main/java/io/gravitee/common/utils/DurationParser.java

```diff
@@ -51,7 +51,7 @@ public class DurationParser {
      */
     public static Duration parse(final String value) {
         Duration duration = null;
-        if (StringUtils.hasLength(value)) {
+        if (value != null) {
```

Comment:
should we test for an empty string?

Reply:
No doesn't matter actually

Reply:
`StringUtils.hasLength` was "working" for both null and empty string. From what I see, now it will throw a `java.time.format.DateTimeParseException` and try the simple format which also fail.
Basically we would have the same behavior with `null` value. So I'm wondering why we would test `null` but not the empty string 🤷‍♂️

Reply:
No it doesn't throw DateTimeParseException. There is a unit test which covers this specific use case.